<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QWidget" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Panel.qrc">:/images/panel2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>276</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>New 4 Digit Password :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePasswordPB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>169</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
color: rgb(64, 0, 225);
font: 10pt &quot;MRT_Sozar&quot;;
</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createNewBankAccountPB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>169</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
font: 10pt &quot;MRT_Sozar&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Create New Bank Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutPB">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="viewBalancePB">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>169</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
color: rgb(0, 0, 0);
font: 10pt &quot;MRT_Sozar&quot;;
</string>
   </property>
   <property name="text">
    <string>View Balance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new4DigitPasswordLE">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>284</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="transferPB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>169</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-decoration: underline;
color: rgb(0, 0, 0);
font: 10pt &quot;MRT_Sozar&quot;;
</string>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewFixedPasswordL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>321</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>New Fixed Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newFixedPasswordLE">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>334</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(158, 255, 174);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change4DigitPasswordsPB">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>224</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;</string>
   </property>
   <property name="text">
    <string>Change 4 Digit Password</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="changePasswordL">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>10</y>
     <width>310</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(64, 0, 255);
font: 800 17pt &quot;Rockwell Extra Bold&quot;;
</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>114</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 10pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Age :</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameL">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>79</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Last Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageST">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(67, 67, 67);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeL">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>114</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 10pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>National Code :</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameST">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>95</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(67, 67, 67);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameST">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>95</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(67, 67, 67);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeST">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(67, 67, 67);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="new4DigitPasswordError">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>new 4 digit password error</string>
   </property>
  </widget>
  <widget class="QLabel" name="newFixedPasswordError">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>new fixed password error</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeFixedPasswordsPB">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;</string>
   </property>
   <property name="text">
    <string>Change Fixed Password</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cardNumberL">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>219</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 12pt &quot;Tw Cen MT&quot;;
</string>
   </property>
   <property name="text">
    <string>Card Number :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cardNumberCB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>229</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 21);
selection-background-color: rgb(36, 36, 36);
background-color: rgb(36, 36, 36);
font: 9pt &quot;MRT_Sozar&quot;;
selection-color: rgb(0, 255, 21);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Panel.qrc"/>
 </resources>
 <connections/>
</ui>
